fiftyone/core/map/mapper.py

```diff
@@ -25,18 +26,19 @@
 import fiftyone.core.config as focc
 import fiftyone.core.map.batcher as fomb
 import fiftyone.core.sample as fos
+import fiftyone.core.utils as fou
 from fiftyone.core.map.typing import SampleCollection
 
 T = TypeVar("T")
 R = TypeVar("R")
+U = TypeVar("U")
```

Comment:
same for declarations in process.py and threading.py also. This will help us in the future
```suggestion
T = TypeVar("T")  # Sample type
R = TypeVar("R")  # Return value type of map_fcn
U = TypeVar("U")  # Return value type of iter_fcn if set - must be same as input type to map_fcn.
```